### PR TITLE
Added tests for database-stocks, refactor tests

### PR DESCRIPTION
**Pull request by Alyks1**
Testing all the Database-stocks functions

### CODE REVIEW

File: database/database-events_test.go [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+				EventID:   event.eventID,
+				Title:     event.eventID + event.eventID,
+				Text:      "",
+				Active:    true,
+				CreatedAt: createdAt,
+			})
+			newEvent, err := suite.dbService.GetEvents(parseTime("2020-12-30T15:00:05Z"))
+			assert.Equal(suite.T(), nil, err)
+
+			// I hate my life
```
Comment (explodingcamera) [inline database/database-events_test.go]:
```suggestion
```

File: database/database-stocks_test.go [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+		err := suite.dbService.DB.Create(&test.stock).Error
+		assert.Equal(suite.T(), nil, err)
+		test.stock.Tick++
+		err = suite.dbService.DB.Create(&test.stock).Error
+		assert.Equal(suite.T(), nil, err)
+		result, err := suite.dbService.GetStocksAtTick(test.tick)
+		assert.Equal(suite.T(), nil, err)
+
+		for j := 0; j < len(test.expectation); j++ {
+			//I hate my life - cant tests equals because of UUID and other stuff
```
Comment (explodingcamera) [inline database/database-stocks_test.go]:
```suggestion
```

File: database/database-stocks_test.go [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+		err := suite.dbService.DB.Create(&test.stock).Error
+		assert.Equal(suite.T(), nil, err)
+		test.stock.Tick++
+		err = suite.dbService.DB.Create(&test.stock).Error
+		assert.Equal(suite.T(), nil, err)
+		result, err := suite.dbService.GetStockMapAtTick(test.tick)
+		assert.Equal(suite.T(), nil, err)
+
+		for j := 0; j < len(test.expectation); j++ {
+			//I hate my life - cant tests equals because of UUID and other stuff
```
Comment (explodingcamera) [inline database/database-stocks_test.go]:
```suggestion
```

File: go.sum [hunk 4]
```diff
@@ -55,6 +55,8 @@ github.com/satori/go.uuid v1.2.0 h1:0uYX9dsZ2yD7q2RtLRtPSdGDWzjeM3TbMJP9utgA0ww=
 github.com/satori/go.uuid v1.2.0/go.mod h1:dA0hQrYB0VpLJoorglMZABFdXlWrHn1NEOzdhQKdks0=
 github.com/senseyeio/duration v0.0.0-20180430131211-7c2a214ada46 h1:Dz0HrI1AtNSGCE8LXLLqoZU4iuOJXPWndenCsZfstA8=
 github.com/senseyeio/duration v0.0.0-20180430131211-7c2a214ada46/go.mod h1:is8FVkzSi7PYLWEXT5MgWhglFsyyiW8ffxAoJqfuFZo=
+github.com/sergi/go-diff v1.1.0 h1:we8PVUC3FE2uYfodKH/nBHMSetSfHDR6scGdBi+erh0=
```
Comment (Arthlight) [inline go.sum:58]: is this really necessary?

**Pull request merged**